packages/worker/tsconfig.json

```diff
@@ -6,7 +6,7 @@
         "composite": true,
         "declaration": true
     },
-    "include": ["src/**/*"],
+    "include": ["src/**/*", "../types/shims/ecs.d.ts"],
```

Comment:
No relative module resolution outside of the package root. It must be included as a dependency `"@promethean/types":"workspace/*"`